src/config.example.hpp

```diff
@@ -12,18 +12,31 @@
 #include "RosClientNode.hpp"
 
 namespace config {
-static const std::string host_url = "ws://localhost:8080";
+static const std::string host_url = "wss://10.0.0.1:8080";
 static const std::string ros_node_name = "robofleet_client";
+static const std::string robot_name = "robot";
 
+// Edit the configuration here to provide the correct topic names.
+static const std::string status_topic = "status";
+static const std::string subscription_topic = "subscriptions";
+static const std::string localization_topic = "localization";
+static const std::string odometry_topic = "odometry/raw";
+static const std::string lidar_2d_topic = "velodyne_2dscan";
+static const std::string left_image_topic = "stereo/left/image_raw/compressed";
+static const std::string right_image_topic = "stereo/right/image_raw/compressed";
+
+// All message types and subscribed topics must be enumerated here.
+// Specializations must also be provided in encode.hpp and decode.hpp
 static void configure_msg_types(RosClientNode& cn) {
-  // All message types and subscribed topics must be enumerated here.
-  // Specializations must also be provided in encode.hpp and decode.hpp
-  cn.register_msg_type<amrl_msgs::RobofleetStatus>("status");
-  cn.register_msg_type<amrl_msgs::RobofleetSubscription>("subscriptions");
-  cn.register_msg_type<amrl_msgs::Localization2DMsg>("localization");
-  cn.register_msg_type<nav_msgs::Odometry>("odometry/raw");
-  cn.register_msg_type<sensor_msgs::LaserScan>("velodyne_2dscan");
-  cn.register_msg_type<sensor_msgs::CompressedImage>(
-      "stereo/left/image_raw/compressed");
+  // These topics are used by webviz. Do not modify this section of the config
```

Comment:
since we can use semantic names for these now, I think we can clean them up a bit